### PR TITLE
update qps and burst for agent

### PR DESCRIPTION
**Pull request by ldpliu**
https://github.com/open-cluster-management/backlog/issues/12776

### CODE REVIEW

File: pkg/utils/rest/mapper.go [hunk 1]
```diff
@@ -65,9 +65,10 @@ func (p *Mapper) Mapper() meta.RESTMapper {
 	return p.mapper
 }
 
-// MappingFor returns the RESTMapping for the Kind given, or the Kind referenced by the resource.
-// Prefers a fully specified GroupVersionResource match. If one is not found, we match on a fully
-// specified GroupVersionKind, or fallback to a match on GroupKind.
+// The resourceOrKindArg should be in the format of {resource}.{version}.{group}, or {resource}.
```
Comment (zlayne) [inline pkg/utils/rest/mapper.go:68]: @ldpliu Hey quick question. Does `ManagedClusterAction` have the same requirement to use either `{resource}.{version}.{group}, or {resource}` or is this just for the `ManagedClusterView`?

Comment (ldpliu) [reply pkg/utils/rest/mapper.go:68]: it's just for the ManagedClusterView

Comment (qiujian16) [reply pkg/utils/rest/mapper.go:68]: i believe `ManagedClusterAction` also needs `{resource}.{version}.{group}`

Comment (ldpliu) [reply pkg/utils/rest/mapper.go:68]: Sorry for that, Managedclusteraction have the same requirement.

**Comment by qiujian16** [comment]
/lgtm

it is still on hold until 2.2.5 is open.

**Comment by anthonyfister** [comment]
@qiujian16 
If you think you have a complete fix, please go ahead and merge to 2.2.5.
We have an RC0 for 2.2.5 and are literally holding the 2.2.5 release for the fix for this BZ (see below).  
Once this is merged, we will build 2.2.5, test and deliver.   
https://bugzilla.redhat.com/show_bug.cgi?id=1964161

**Pull request merged**